app/models/slack_user.rb

```diff
@@ -14,6 +14,8 @@ class SlackUser < ApplicationRecord
   scope :bots, -> { where(is_bot: true) }
   scope :human, -> { where(is_bot: false) }
   scope :available, -> { extant.human }
+  scope :active, -> { available.where.not(last_message_at: nil) }
```

Comment:
I think this should be more like available.where(SlackUser.arel_table[:last_message_at].gt 1.month.ago)